ansible/roles/fanout/tasks/sonic/fanout_sonic_202405.yml

```diff
@@ -0,0 +1 @@
+fanout_sonic_202311.yml
```

Comment:
Don't think we should inherit all fanout_sonic_202311.yml for 202405, although the code maybe the same, we can use cherry-pick to sync

Reply:
Hi @nhe-NV , Do you have some comment?

Reply:
Actaully I don't think duplicate code is a good option, at this moment, the content are identical, why we cna use the same file, or use the soft link to point to the same content.  In the future if we have to differeciate the content in different branch, then we can split, before that, I think it current implement is better.

Reply:
I think it's OK to use soft link so far.
In the future, if 202405 and 202311 are getting different, the fix would be pretty easy.